### DISCUSSION TITLE
How do I load 3rd party scripts in Elementor Editor in WordPress

### DISCUSSION THREAD

**Discussion by bstrovincent** [elementor/elementor#24619] [Developers Community]
### Start a discussion

I built a custom carousel widget using Slick slider JS and have no issues on the frontend but when editing the page using Elementor, the css is completely off. Elementor seems to not enqueue 3rd party scripts in the editor. Is there some kind of hook that allows me to enqueue the necessary scripts in editor?

**Comment by m1ga** [comment]
https://developers.elementor.com/docs/scripts-styles/editor-scripts/

just keep in mind that your elements might not be ready yet when you register for those events.

edit: https://developers.elementor.com/docs/scripts-styles/preview-styles/ with a setInterval works.
